nuxeo-coldstorage/src/test/java/org/nuxeo/coldstorage/operations/AbstractTestRestoreFromColdStorageOperation.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * (C) Copyright 2021 Nuxeo (http://nuxeo.com/) and others.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * Contributors:
```

Comment:
Could you check the javadoc. You are missing some extra lines before `Contributors`